test/cross-chain/cross-chain.js

```diff
@@ -148,4 +380,585 @@ contract("Cross-chain", () => {
+    });
+
+    it("if bridge is broken, bridging skill transaction doesn't revert (allowing e.g. domains to be created)", async () => {
+      let tx = await foreignBridge.setBridgeEnabled(false);
+      await tx.wait();
+      const skillCount = await foreignColonyNetwork.getSkillCount();
+
+      tx = await foreignColonyNetwork.bridgeSkill(skillCount, { gasLimit: 1000000 });
+      let receipt = await tx.wait();
+      expect(receipt.status).to.equal(1);
+
+      tx = await foreignColony["addDomain(uint256,uint256,uint256)"](1, ethers.BigNumber.from(2).pow(256).sub(1), 1);
+      receipt = await tx.wait();
+
+      let events = receipt.logs.map(function (log) {
+        try {
+          return foreignColonyNetwork.interface.parseLog(log);
+        } catch (e) {
+          // Return nothing
+        }
+        return null;
+      });
+      events = events.filter((x) => x != null && x.eventFragment.name === "SkillCreationStored");
+      expect(events.length).to.equal(1);
+      const event = events[0];
+      expect(event.args[0].toString()).to.equal(skillCount.add(1).toString());
+    });
+
+    it("colony root local skill structures end up the same on both chains", async () => {
+      const homeColonyRootLocalSkillId = await homeColony.getRootLocalSkill();
+      let homeColonyRootLocalSkill = await homeColonyNetwork.getSkill(homeColonyRootLocalSkillId);
+
+      const foreignColonyRootLocalSkillId = await foreignColony.getRootLocalSkill();
+      let foreignColonyRootLocalSkill = await foreignColonyNetwork.getSkill(foreignColonyRootLocalSkillId);
+
+      expect(homeColonyRootLocalSkill.nParents.toString()).to.equal(foreignColonyRootLocalSkill.nParents.toString());
+      expect(homeColonyRootLocalSkill.nChildren.toString()).to.equal(foreignColonyRootLocalSkill.nChildren.toString());
+
+      let tx = await homeColony.addLocalSkill();
+      await tx.wait();
+
+      const p = getPromiseForNextBridgedTransaction();
+      tx = await foreignColony.addLocalSkill();
+      await tx.wait();
+      await p;
+      homeColonyRootLocalSkill = await homeColonyNetwork.getSkill(homeColonyRootLocalSkillId);
+      foreignColonyRootLocalSkill = await foreignColonyNetwork.getSkill(foreignColonyRootLocalSkillId);
+
+      expect(homeColonyRootLocalSkill.nParents.toString()).to.equal(foreignColonyRootLocalSkill.nParents.toString());
+      expect(homeColonyRootLocalSkill.nChildren.toString()).to.equal(foreignColonyRootLocalSkill.nChildren.toString());
+
+      let zeroSkill = await foreignColonyNetwork.getSkill(ethers.BigNumber.from(foreignChainId).mul(ethers.BigNumber.from(2).pow(128)));
+      expect(zeroSkill.nChildren.toNumber()).to.equal(0);
+
+      zeroSkill = await homeColonyNetwork.getSkill(ethers.BigNumber.from(foreignChainId).mul(ethers.BigNumber.from(2).pow(128)));
+      expect(zeroSkill.nChildren.toNumber()).to.equal(0);
+
+      zeroSkill = await homeColonyNetwork.getSkill(0);
+      expect(zeroSkill.nChildren.toNumber()).to.equal(0);
```

Comment:
Is this a meaningful test?

Reply:
Yes, because until [these changes](https://github.com/JoinColony/colonyNetwork/commit/8e117b1157badc11e5de62a608adea0281a238c7) it wasn't true.